testgrid/config/config_test.go

```diff
@@ -145,6 +145,14 @@ func TestConfig(t *testing.T) {
 		}
 	}
 
+	for _, dashboardGroup := range config.DashboardGroups {
```

Comment:
Can you also add tests to 1) ensure no dashboard names are duplicated in a group, and 2) there aren't duplicated dashboard group names?